src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -21,7 +24,11 @@ public DelayedDeliverySettings DisableTimeoutManager()
         }
 
         /// <summary>
-        /// TODO: Fill this out
+        /// Tells the endpoint that all other endpoints also support native delays.
+        /// <para>
+        /// If this is called, the endpoint will no longer bind destination addresses to the delay
```

Comment:
@WojcikMike Yeah, I can see how going for something simpler makes sense. I'll push up a change.